src/GitHubExtension/Providers/RepositoryProvider.cs

```diff
@@ -239,12 +239,12 @@ public IAsyncOperation<ProviderOperationResult> CloneRepositoryAsync(IRepository
             catch (LibGit2Sharp.LibGit2SharpException libGitTwoException)
             {
                 Providers.Log.Logger()?.ReportError("DevHomeRepository", $"Either no logged in account has access to this repo, or the repo can't be found", libGitTwoException);
-                return new ProviderOperationResult(ProviderOperationStatus.Failure, libGitTwoException, "LigGit2 threw an exception", "LibGit2 Threw an exception");
+                return new ProviderOperationResult(ProviderOperationStatus.Failure, libGitTwoException, "LibGit2 threw an exception", "LibGit2 threw an exception");
             }
             catch (Exception e)
             {
                 Providers.Log.Logger()?.ReportError("DevHomeRepository", "Could not clone the repository", e);
-                return new ProviderOperationResult(ProviderOperationStatus.Failure, e, "Something happened when cloning the repo", "something happened when cloning the repo");
+                return new ProviderOperationResult(ProviderOperationStatus.Failure, e, "Something happened when cloning the repo", "Something happened when cloning the repo");

```

Comment:
I know this is a bit nitpicky, but the entire PR/issue is basically a nitpick, so I think it's warranted. If we're going to fix this sort of issue, we should make whatever is touched as correct as possible, so we don't have to fix it again later.

Two issues I see here:
1) Use of abbreviations.  The abbreviation "repo" is used instead of "repository. Abbreviations and slang are to be avoided. Yes, I know this is all over the code, but if going to change something make it right as we go. 
2) All the error messages touched in this PR are not complete sentences. Namely just add a period at the end.
 
https://learn.microsoft.com/en-us/windows/win32/debug/error-message-guidelines

Reply:
Thanks, I'll make these corrections. 

BTW: I agree that this PR is a nitpick, But I actually did spent several minutes trying to figure out / web searching what "LitGit2" meant in the error dialogue I received before I realized it was a typo (I know, not my brightest moment) - hence the PR.

Reply:
Updated the PR. I checked all messages for spelling errors, punctuation, abbreviations and made a few small changes to improve clarity.